src/Generator/Entity/ModelEntity.cs

```diff
@@ -16,8 +16,10 @@ internal ModelEntity(DTInterfaceInfo interfaceInfo, ModelGeneratorOptions option
         Name = GetClassName(interfaceInfo.Id);
         Parent = interfaceInfo.Extends.Count() > 0 ? GetClassName(interfaceInfo.Extends.First().Id) : nameof(BasicDigitalTwin);
         FileDirectory = ExtractDirectory(ModelId);
-        var contents = interfaceInfo.Contents.Select(c => c.Value).Where(c => c.Id.Labels.Contains(Name));
-        Content.AddRange(contents.Select(CreateProperty));
+        var contents = interfaceInfo.Contents.Select(c => c.Value).Where(c => c.Id.Labels.Contains(Name) && c is not DTCommandInfo);
+        PropertyContent.AddRange(contents.Select(CreateProperty));
+        var commands = interfaceInfo.Contents.Select(c => c.Value).Where(c => c.Id.Labels.Contains(Name) && c is DTCommandInfo);
+        CommandContent.AddRange(commands.Select((contentInfo) => new Command((DTCommandInfo)contentInfo, Name, Options)));
     }
```

Comment:
Some thoughts...

```suggestion
        var properties = NEEDS_A_DECENT_NAME<DTPropertyInfo>(interfaceInfo);
        PropertyContent.AddRange(properties.Select(CreateProperty));
        var relationships = NEEDS_A_DECENT_NAME<DTRelationshipInfo>(interfaceInfo);
        PropertyContent.AddRange(relationships.Select(CreateProperty));
        var commands = NEEDS_A_DECENT_NAME<DTCommandInfo>(interfaceInfo);
        CommandContent.AddRange(commands.Select((contentInfo) => new Command((DTCommandInfo)contentInfo, Name, Options)));
    }

    // ... protected methods go here ...

    // This method *could* probably exist in a parent class, but we can move it "up" later on if there's a need...
    private IEnumerable<DTContentInfo> NEEDS_A_DECENT_NAME<T>(DTInterfaceInfo interfaceInfo) where T : DTContentInfo // <-- I think that's the right type constraint...
    {
        return interfaceInfo.Contents.Select(c => c.Value).Where(c => c.Id.Labels.Contains(Name) && c is T);
    }
```
Maybe `NEEDS_A_DECENT_NAME` could be `SelectContentInfo`?

I'd also consider creating a `CreateCommand()` in the `ClassEntity` class (matching the pattern used with `CreateProperty()`):
```csharp
    protected Command CreateCommand(DTContentInfo content)
    {
        if (content is DTCommandInfo command)
        {
            return new Command(command, Name, Options);
        }

        throw new Exception($"Unsupported content type: {content.EntityKind}");
    }
```
(Nilay's PR #24 introduces `UnsupportedContentTypeException` that should be used instead, but maybe we address it in that PR if this one goes first...)